pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -649,7 +649,7 @@ func (p *criStatsProvider) getAndUpdateContainerUsageNanoCores(stats *runtimeapi
 		defer p.mutex.Unlock()
 
 		cached, ok := p.cpuUsageCache[id]
-		if !ok || cached.stats.UsageCoreNanoSeconds == nil {
+		if !ok || cached.stats.UsageCoreNanoSeconds == nil || stats.Cpu.UsageCoreNanoSeconds.Value < cached.stats.UsageCoreNanoSeconds.Value {
```

Comment:
Just to confirm - is the only way that `stats.Cpu.UsageCoreNanoSeconds < cached.stats.UsageCoreNanoSeconds` if `cpuacct` is reset?

If that's the case, could we use a descriptive variable name here? For example,
```
cpuAcctReset := stats.Cpu.UsageCoreNanoSeconds.Value < ...
if !ok || ... || cpuAcctReset {
  ...
}
```